### PR TITLE
Docs/0.1 Pre-release

### LINKED ISSUE #149: Check Snowflake documentation for release
We need to ensure that all documentation is polished.

This includes but is not limited to
* Examples
* All public API's should documented with doctest where applicable

### PR DESCRIPTION
**Pull request by plerouxanyon**
Closes #149

### CODE REVIEW

**Review by fSosaRey — CHANGES_REQUESTED** [review]
in getting_started.md, it would be nice to include a Bloch sphere rendering of a Ket.

**Comment by plerouxanyon** [comment]
> in getting_started.md, it would be nice to include a Bloch sphere rendering of a Ket.

The getting_started.md example is of a Bell state and not representable on a Bloch sphere. Do you have a suggestion on how to incorporate it?

File: docs/src/getting_started.md [hunk 1]
```diff
@@ -1,5 +1,50 @@
...
+### Installing the Snowflake package
+Snowflake is still in pre-release phase. Therefore, and for the time being, we recommand installing it by checking out the `main` branch from github. This can be achieved by typing the following commands in the Julia REPL:
+
+```julia
+import Pkg
+Pkg.add(url="https://github.com/anyonlabs/Snowflake.jl", rev="main")
+```
+This will add the Snowflake  package to the current [Julia Environment](https://pkgdocs.julialang.org/v1/environments/).
+
+Once `Snowflake.jl` is released, you can install the latest release using the following command:
```
[the remark this answers is not in the document]
Comment (plerouxanyon) [reply docs/src/getting_started.md]: I've added a warning for the user mentioning that it will pull an much older version.

**Review by alireza-najafi — APPROVED** [review]
For some reason, I've lost my edits. I will approve this merge request and then try to start a new documentation branch.

**Pull request merged**